setup.py

```diff
@@ -42,6 +42,9 @@
           'setuptools',
           'plone.rest >= 1.0a6',  # json renderer moved to plone.restapi
           'PyJWT',
+          'five.intid',
+          'plone.api',
```

Comment:
Then, until the discussion is closed, we should change the `@principals` endpoint not to use plone.api: https://github.com/plone/plone.restapi/blob/master/src/plone/restapi/services/principals/get.py#L45